tests/suites/test_suite_rsa.data

```diff
@@ -386,18 +386,25 @@ RSA Public (Data = 0)
 mbedtls_rsa_public:"00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000":2048:"b38ac65c8141f7f5c96e14470e851936a67bf94cc6821a39ac12c05f7c0b06d9e6ddba2224703b02e25f31452f9c4a8417b62675fdc6df46b94813bc7b9769a892c482b830bfe0ad42e46668ace68903617faf6681f4babf1cc8e4b0420d3c7f61dc45434c6b54e2c3ee0fc07908509d79c9826e673bf8363255adb0add2401039a7bcd1b4ecf0fbe6ec8369d2da486eec59559dd1d54c9b24190965eafbdab203b35255765261cd0909acf93c3b8b8428cbb448de4715d1b813d0c94829c229543d391ce0adab5351f97a3810c1f73d7b1458b97daed4209c50e16d064d2d5bfda8c23893d755222793146d0a78c3d64f35549141486c3b0961a7b4c1a2034f":"3":"00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000":0
 
 RSA Generate Key - 128bit key
+depends_on:MBEDTLS_RSA_MIN_KEY_SIZE == 128
 mbedtls_rsa_gen_key:128:3:0
 
+RSA Generate Key - 128bit key (Less than minimum size)
+depends_on:MBEDTLS_RSA_MIN_KEY_SIZE > 128
+mbedtls_rsa_gen_key:128:3:MBEDTLS_ERR_RSA_BAD_INPUT_DATA
```

Comment:
Rather parametrize the test cases on `MBEDTLS_RSA_MIN_KEY_SIZE`:

```suggestion
RSA Generate Key - minimum size
mbedtls_rsa_gen_key:MBEDTLS_RSA_MIN_KEY_SIZE:3:0

RSA Generate Key - less than minimum size
depends_on:MBEDTLS_RSA_MIN_KEY_SIZE - 2 >= 128
mbedtls_rsa_gen_key:MBEDTLS_RSA_MIN_KEY_SIZE - 2:3:MBEDTLS_ERR_RSA_BAD_INPUT_DATA
```

Reply:
Done.

Reply:
Ah, yes. Dependencies have to be of the form `NAME`, `!NAME` or `NAME OP INTEGER` where OP is a comparison operator. (I don't think there's really a good reason for that.)